### PR TITLE
Code Reviews

### PR DESCRIPTION
**Pull request by JouharaAlwardi**
I add the following points to the Code Review task: 

- How to perform a Tool-Assisted code review
- How to Make code reviews better
- Code Review Checklist

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -18,6 +18,60 @@ from getting into your project at an early stage of the software development pro
...
+
+
+### 2. How to Make Code Reviews Better
+
+---
+One of the ways to make code reviews better is through tone of the review which can greatly influence morale within teams. 
+
+**Good code reviews**: is when a reviewr has an open mind and not judge and make opinionated statements. These views should be clarified instead of forcing correction. 
+
+**Better code reviews**: the reviewr is more empathetic and takes time to go through every single line of the code. They have postive feedback and  applaud nice solutions.
```
Comment (xtreger) [inline README.md]: Small spelling mistake here.

File: README.md [hunk 2]
```diff
@@ -18,6 +18,60 @@ from getting into your project at an early stage of the software development pro
...
++ Examples of code reviews
+
+&nbsp;
+
+
+### 1. How to perform a Tool-Assisted code review
+
+---
+A tool-assisted code review process involves the use of a specialized tool to facilitate the process of code review such as Crucible, Review Board and GitHub. A tool generally helps reviewrs with the following tasks:
+
```
Comment (xtreger) [inline README.md:31]: Similar mistake here.

File: README.md [hunk 3]
```diff
@@ -18,6 +18,60 @@ from getting into your project at an early stage of the software development pro
...
+&nbsp;
+
+
+### 2. How to Make Code Reviews Better
+
+---
+One of the ways to make code reviews better is through tone of the review which can greatly influence morale within teams. 
+
+**Good code reviews**: is when a reviewr has an open mind and not judge and make opinionated statements. These views should be clarified instead of forcing correction.
```
Comment (xtreger) [inline README.md]: Similar here.

File: README.md (hunk 1 above)
Comment (xtreger) [reply README.md]: two here actually, another in the second sentence

File: README.md [hunk 4]
```diff
@@ -18,6 +18,60 @@ from getting into your project at an early stage of the software development pro
...
+
+&nbsp;
+
+
+### 3. Code Review Checklist
+
+---
+The following code review checklist gives an idea about the various aspects you need to consider while reviewing the code:
+
+**1. Non-functional requirments:** The code should meet the non-functional requirments such as Maintainability (Supportability) where  The application should require the least amount of effort to support in near future. It should be easy to identify to fix any defects.
```
Comment (xtreger) [inline README.md]: Another here.

Comment (xtreger) [reply README.md]: two actually.

File: README.md [hunk 5]
```diff
@@ -18,6 +18,60 @@ from getting into your project at an early stage of the software development pro
...
+
+**2. Code formatting:** includes the use of alignments, proper naming connventions and it should fit the standard 14 inch laptop screen.
```
Comment (xtreger) [inline README.md]: One here.

**Review by diyarenny — APPROVED** [review]
I can see that you fixed all the errors as highlighted, approving PR.

**Review by xtreger — APPROVED** [review]
Looks good, approving PR.

**Review by oisinallen99 — APPROVED** [review]
looks good

**Pull request merged**